03Lesson/01Project/index.html

```diff
@@ -1,9 +1,97 @@
 <!DOCTYPE html>
 <html>
+<meta charset="utf-8">
 <head>
-  <title></title>
+  <title>Intel</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+
+  <section class="blue" class="dev_1">

```

Comment:
Duplicate class attribute is not be allowed. Should be mentioned in w3c validator.